### PR TITLE
Update pingpong.rs example

### PR DESCRIPTION
**Pull request by Alex6323**
* update example
* change reconnect interval default to 60s
* use Duration instead of u64
* use Port in NetworkConfig

### CODE REVIEW

File: bee-network/src/timing.rs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pub(crate) const DEFAULT_RECONNECT_INTERVAL: Seconds = Seconds(60);
```
Comment (thibault-martinez) [inline bee-network/src/timing.rs:1]: This is a helper file, I don't think this default should be here.

Comment (thibault-martinez) [reply bee-network/src/timing.rs:1]: Also, I feel like this file coule be in bee-common.

File: bee-network/src/config.rs [hunk 2]
```diff
@@ -59,17 +62,18 @@ impl NetworkConfigBuilder {
...
 }
 
 /// Network configuration.
 #[derive(Clone, Copy, Debug)]
 pub struct NetworkConfig {
+    // TODO: use Port instead of u16
     pub(crate) binding_port: u16,
     pub(crate) binding_addr: IpAddr,
-    pub(crate) reconnect_interval: u64,
+    pub(crate) reconnect_interval: Seconds,
```
Comment (thibault-martinez) [inline bee-network/src/config.rs:76]: Why not directly using a Duration btw ?

Comment (Alex6323) [reply bee-network/src/config.rs]: Really good point. I don't know why I wanted something own. This is one of those situations where you're like: "What the hell was I thinking?" :joy: Maybe I though that `Seconds` would be better readable.

File: bee-network/src/timing.rs (hunk 1 above)
Comment (Alex6323) [reply bee-network/src/timing.rs]: Removed it entirely.

**Pull request merged**